packages/mobile/src/home/SendOrRequestBar.test.tsx

```diff
@@ -37,6 +37,17 @@ describe('SendOrRequestBar', () => {
       </Provider>
     )
     expect(getByTestId('SendOrRequestBar/SendButton')).toBeDisabled()
-    expect(getByTestId('SendOrRequestBar/RequestButton')).toBeDisabled()
+  })
+
+  it('request button enabled when there are no tokens', () => {
+    const store = createMockStore({
+      tokens: {},
+    })
+    const { getByTestId } = render(
+      <Provider store={store}>
+        <SendOrRequestBar />
+      </Provider>
+    )
+    expect(getByTestId('SendOrRequestBar/RequestButton')).not.toBeDisabled()
```

Comment:
I think it would have been fine to put this line in the previous test, all the setup and everything is the same than in it, so in the spirit of not repeating code I'd have done that. This is fine too of course, no need to change it if you prefer it this way, just sharing my thoughts.